proc.go

```diff
@@ -293,7 +289,7 @@ func (p *Proc) draw(e system.FrameEvent) {
 	paint.Fill(ops, clr)
 
 	p.Draw()
-
+	p.incFrameCount()
```

Comment:
I'd move this to after the frame has been displayed.
(or even make it a `defer` and move it at the top of the `draw` func).